### PR TITLE
Project 3: Style core elements

### PR DESCRIPTION
**Pull request by crgc**
### Features
- Used plain CSS (no framework) to style the app according to the required layout

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+main {
+  width: 1200px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
```
Comment (mimipeshy) [inline css/style.css]: - [x]   Let's keep our code DRY. `display: flex` and `flex-direction: column` has been repeated more than once. To solve this you can group all classes with the same attribute together and give it one attribute eg
```
main,
.add-btn {
    display:flex
}
```

File: index.html [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
+         <h2>Add a new book</h2>
+         <input class="input-title" type="text" placeholder="Title">
+         <input class="input-author" type="text" placeholder="Author">
+         <div class="add-btn">
+            <button id="add-btn" type="button">Add</button>
+         </div>
+         </div>
+      </main>
+      <script type="module" src="js/main.js"></script>
+      <!-- <script src="js/_main_.js"></script> -->
```
Comment (mimipeshy) [inline index.html]: - [x] Kindly lets remove commented out code so that we keep the codebase clean

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi Team @crgc and @eri8-9 👋🏾 👋🏾  

Good job so far!

#### Highlights

- Great UI design ✔️ 
- Good use of proper git-flow ✔️ 
- Good commit messages ✔️ 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
------

**Review by voscarmv — APPROVED** [review]
Hi @crgc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**